src/expressions.rst

```diff
@@ -369,28 +369,110 @@ The following :t:`[expression]s` are :t:`[place expression]s`:
   expression`,
 
 * :dp:`fls_ya05djl1d154`
-  :t:`[Path expression]s` that resolve to a :t:`static` or a :t:`variable`.
+  :t:`[Path expression]s` that resolve to a :t:`static` or a :t:`local
+  variable`.
+
+:dp:`fls_ku38h562vfyl`
+A :t:`mutable place expression` is a :t:`place expression` that allows
```

Comment:
`mutable place expression` is already defined in the Glossary. It would be nice to use the same definition, which is

A mutable place expression is a place expression whose memory location can be modified.